src/tests/validity/mv.ab

```diff
@@ -0,0 +1,5 @@
+unsafe $cd /tmp$
+unsafe $touch a$
```

Comment:
that's no way to create a temporary file. `mktemp` is a part of gnu coreutils and provides a nice way to create it

Reply:
other tests uses touch to create a temporary file, maybe we need to align all of them for that in another ticket